src/server/services/wcs/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+
+########################################################
+# Files
+
+SET (wcs_SRCS
+  qgswcs.cpp
+  qgswcsutils.cpp
+  qgswcsgetcapabilities.cpp
+  qgswcsdescribecoverage.cpp
+  qgswcsgetcoverage.cpp
+)
+
+########################################################
+# Build
+
+ADD_LIBRARY (wcs MODULE ${wcs_SRCS})
+
+
+INCLUDE_DIRECTORIES(SYSTEM
+  ${GDAL_INCLUDE_DIR}
+  ${GEOS_INCLUDE_DIR}
+  ${PROJ_INCLUDE_DIR}
+  ${POSTGRES_INCLUDE_DIR}
+)
+
+INCLUDE_DIRECTORIES(
+  ${CMAKE_BINARY_DIR}/src/core
+  ${CMAKE_BINARY_DIR}/src/gui
+  ${CMAKE_BINARY_DIR}/src/python
+  ${CMAKE_BINARY_DIR}/src/analysis
+  ${CMAKE_BINARY_DIR}/src/server
+  ${CMAKE_CURRENT_BINARY_DIR}
+  ../../../core 
+  ../../../core/dxf
+  ../../../core/geometry 
+  ../../../core/raster
+  ../../../core/symbology-ng
+  ../../../core/composer
+  ../../../core/layertree
+  ../../../gui 
```

Comment:
This is only needed for wms which have a dependency on QgsEditorWidgetRegistry. Removed for wcs and wfs.